src/components/progress-circle/progress_circle.ts

```diff
@@ -0,0 +1,123 @@
+import {
+  Component,
+  ViewEncapsulation,
+  ElementRef,
+  Attribute,
+  Input,
+  HostBinding
+} from 'angular2/core';
+import {isPresent, CONST} from 'angular2/src/facade/lang';
+import {OneOf} from '../../core/annotations/one-of';
+
+
+// TODO(josephperrott): Benchpress tests.
+
+/** Display modes of Progress Circle */
+@CONST()
+class ProgressMode {
+  @CONST() static DETERMINATE = 'determinate';
+  @CONST() static INDETERMINATE = 'indeterminate';
+}
+
+
+/**
+ * <md-progress-circle> component.
+ */
+@Component({
+  selector: 'md-progress-circle',
+  host: {
+    'role': 'progressbar',
+    'aria-valuemin': '0',
+    'aria-valuemax': '100',
+  },
+  templateUrl: './components/progress-circle/progress_circle.html',
+  styleUrls: ['./components/progress-circle/progress-circle.css'],
```

Comment:
I forgot this earlier, but can you add this to the component config:
```ts
changeDetection: ChangeDetectionStrategy.OnPush,
```
Here and on `md-spinner`

(see #67 for details)